docs/Custom tracking events.md

```diff
@@ -0,0 +1,92 @@
+# Create your custom tracking event
+If none of the available tracking events matches your needs you can also extend `TrackingEvent` and create your own. 
```

Comment:
This way we can have links.

```suggestion
If none of the available tracking events matches your needs you can also extend `VSDKTrackingEvent` and create your own. 
```

Before:
![Screenshot 2019-07-23 at 15 17 21](https://user-images.githubusercontent.com/9095073/61719536-03461880-ad5d-11e9-80c6-9e3c761cff76.png)

After:
![Screenshot 2019-07-23 at 15 16 39](https://user-images.githubusercontent.com/9095073/61719541-080acc80-ad5d-11e9-9588-4eb341e67c14.png)

Link: docs/docs_page/custom-tracking-events.html

Reply:
Cool feature 😮 Did not notice that. See 078a496